sources/packages/backend/apps/api/src/auth/roles.enum.ts

```diff
@@ -30,5 +30,6 @@ export enum Role {
   InstitutionCreateNote = "institution-create-note",
   InstitutionApproveDeclineDesignation = "institution-approve-decline-designation",
   InstitutionApproveDeclineOfferingChanges = "institution-approve-decline-offering-changes",
+  InstitutionApproveDeclineApplicationOfferingChangeRequest = "institution-approve-decline-application-offering-change-request",
```

Comment:
Was the new entry updated in the spreadsheet?
https://bcgov.sharepoint.com/:x:/r/teams/03177/Shared%20Documents/General/Analysis/SIMS%20User%20Profiles%20and%20Access%20Permissions/User%20Profile%20Matrix_MinistryUserOnly_2022%20-%20v2.xlsx?d=wfbaf90a95c0c4035bf5c7858ced24a18&csf=1&web=1&e=Knv0oV

Reply:
Added it now. Sorry, I wasn't able to find the sheet earlier.